com.cafeteria/src/main/java/client/handler/AdminCommandHandler.java

```diff
@@ -0,0 +1,86 @@
+package client.handler;
+
+import java.util.Scanner;
+
+public class AdminCommandHandler {
+    public static String getAdminCommand(int choice, Scanner scanner) {
+        switch (choice) {
+            case 1:
+                return getAddUserCommand(scanner);
+            case 2:
+                return "ADMIN VIEW_USERS";
+            case 3:
+                return "ADMIN VIEW_MENU";
+            case 4:
+                return getAddMenuItemCommand(scanner);
+            case 5:
+                return getUpdateMenuItemCommand(scanner);
+            case 6:
+                return deleteMenuItemCommand(scanner);
+            case 7:
+                return "LOGOUT";
+            default:
+                return null;
+        }
+    }
+
+    private static String getAddUserCommand(Scanner scanner) {
+        System.out.println("Enter the username:");
+        String newUsername = scanner.nextLine();
+        System.out.println("Enter the password:");
+        String newPassword = scanner.nextLine();
+        System.out.println("Enter the user's role (1 for Admin, 2 for Chef, 3 for Employee):");
+        int roleId = Integer.parseInt(scanner.nextLine());
+        return String.format("ADMIN REGISTER_USER,%s,%s,%d", newUsername, newPassword, roleId);
+    }
+
+    private static String getAddMenuItemCommand(Scanner scanner) {
```

Comment:
appending command word to this function name is not giving clarity , it  sounds like we are having some command written here.